src/main/groovy/org/xitikit/iqserver/gradle/IqServerExtension.groovy

```diff
@@ -6,10 +6,14 @@ class IqServerExtension {
 
     private File outputDir
 
-    private final IqServerData iqServerData = new IqServerData()
+    private final IqServerData iqServerData
+
+    IqServerExtension(final IqServerData iqServerData) {
+        this.iqServerData = iqServerData
+    }
 
     void iqServerData(Action<? super IqServerData> action) {
-        action.execute(iqServerData)
+        action.execute(this.iqServerData)
```

Comment:
What's the purpose of adding "this."? Is there a scoping issue with groovy here?

Reply:
I think it's just a force of habit from writing Java. No scoping issue that I'm aware of.